.github/workflows/default.yaml

```
@@ -127,7 +139,8 @@ jobs:
 
       - name: Run build on Linux
         if: runner.os == 'Linux'
-        run: ./test-builds.sh ${{ matrix.layer }}
+        run: |
+          ./test-builds.sh --cleanup ${{ matrix.layer.layer }}
```

Copying from a seemingly forgotten (and now outdated) [change request](https://github.com/squid-cache/squid/pull/1812#discussion_r1597260377):

```suggestion
        run: ./test-builds.sh ${{ matrix.layer.layer }}
```

.... or, if the layer.layer change request is addressed:

```suggestion
        run: ./test-builds.sh ${{ matrix.layer.name }}
```

```suggestion
        run: ./test-builds.sh ${{ matrix.layer.test-builds-parameter }}
```